bindings/python/examples/iio_stream.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
I believe it is a good practice for the script to be able to detect if it is being run as script or imported as a module. Maybe others can offer more suggestions on this.
https://docs.python.org/3/library/__main__.html

Reply:
Solved.